syntax = "proto3";
package com.panelsw.ca;
option optimize_for = LITE_RUNTIME;

message TaskSchedulerDetails{

	enum Action
	{
		Create = 0;
		Delete = 1;
		Backup = 2;
		Restore = 3;
	}


	bytes name = 1;
	bytes taskXml = 2;
	bytes backupFile = 3;
	Action action = 4;
	bytes user = 5;
	bytes password = 6;
}
